moveit_ros/planning/planning_scene_monitor/src/current_state_monitor.cpp

```diff
@@ -333,68 +366,69 @@ bool planning_scene_monitor::CurrentStateMonitor::waitForCompleteState(const std
   return ok;
 }
 
-void planning_scene_monitor::CurrentStateMonitor::jointStateCallback(const sensor_msgs::JointStateConstPtr& joint_state)
+void planning_scene_monitor::CurrentStateMonitor::jointStateCallback(
```

Comment:
The changes starting below are pretty hard to review. Is one of the branches not clang-formatted here?